modules/src/core/ics04_channel/context.rs

```diff
@@ -71,10 +72,28 @@ pub trait ChannelReader {
     /// Returns the current timestamp of the local chain.
     fn host_timestamp(&self) -> Timestamp;
 
+    /// Returns the time when the client state of the given identifier is stored
+    fn processed_time(&self, client_id: &ClientId, height: Height) -> Result<Timestamp, Error>;
+
+    /// Returns the height when the client state of the given identifier is stored
+    fn processed_height(&self, client_id: &ClientId, height: Height) -> Result<Height, Error>;
+
```

Comment:
For these two methods, I would find it more accurate if they were called `client_update_time` and `client_update_height`. The reason is that we are in the `ChannelReader` here, and other methods follow a similar convention, for example: `client_consensus_state` or `connection_end`.

Reply:
8dcb33d3db390346c9b96ec98f84fdfa63adb618